assets/app/views/_deployment-config-metadata.html

```diff
@@ -1,11 +1,9 @@
 <div ng-if="deploymentConfigId != ''" class="metadata">
   <span>Created from deployment config {{deploymentConfigId}}</span>
-  <span class="pficon-layered" ng-if="!exists" data-toggle="tooltip" data-placement="right" title="Deployment config no longer exists" style="cursor: help;">
-    <span class="pficon pficon-warning-triangle"></span>
-    <span class="pficon pficon-warning-exclamation"></span>
+  <span ng-if="!exists" data-toggle="tooltip" data-placement="right" title="Deployment config no longer exists" style="cursor: help;">
+    <span class="pficon pficon-error-circle-o"></span>
```

Comment:
any reason to not just move the classes to the parent span? the only reason it was nested before was because it was a layered icon